tests/tracer/test_ext.py

```diff
@@ -24,4 +24,6 @@ def _updateenv(monkeypatch, env):
 @pytest.mark.parametrize("name,environment,tags", _ci_fixtures())
 def test_ci_providers(monkeypatch, name, environment, tags):
     _updateenv(monkeypatch, environment)
-    assert tags == ci.tags(environment), "wrong tags in {0} for {1}".format(name, environment)
+    extracted_tags = ci.tags(environment)
+    for key, value in tags.items():
+        assert extracted_tags[key] == value, "wrong tags in {0} for {1}".format(name, environment)
```

Comment:
We should probably expand this a bit to also assert on the expected tags regardless of the fixtures used, since how do we know from this test case that our `git.commit.author.name` extraction is correct?

Reply:
or maybe we can combine the shared test fixtures with some ones just for us?

Reply:
Part of the issue is that the spec files only provide environment variables that we'd normally get from each individual CI provider, which doesn't include git data like commit author date or committer info. Does it make sense then to mock the git shell command to provide test values?

Reply:
ah, ok, so this test will never test for those then?

this should be ok then, since we are asserting that all expected tags from the fixture is present in the our extracted tags from the env variables.

and then that also makes complete sense to move the git tagging to another test case and not mix them 👍🏻